app/experimenter/experiments/api/v5/queries.py

```diff
@@ -88,11 +86,11 @@ class Query(graphene.ObjectType):
         description="Nimbus Configuration Data for front-end usage.",
     )
 
-    def resolve_experiments(root, info, offset=0, limit=20, status=None):
+    def resolve_experiments(root, info, status=None):
         q = NimbusExperiment.objects.all()
         if status:
             q = q.filter(status=status)
-        return q[offset:limit]
+        return q
```

Comment:
Looking at https://github.com/mozilla/experimenter/blob/main/app/experimenter/nimbus-ui/src/gql/experiments.ts it looks like we also don't filter on status?

Reply:
Heh, you're right, client [does it](https://github.com/mozilla/experimenter/blob/main/app/experimenter/nimbus-ui/src/components/PageHome/index.tsx#L36:L36). Want me to 🪓 here?

Reply:
![rfixXhr](https://user-images.githubusercontent.com/119884/107578643-ff3d4e00-6bc1-11eb-8ec1-491cf0e29eb0.gif)

Reply:
You know it would also be nice if we went through those GQL queries and nuked all the fields that are queried that aren't actually being used on the frontend, and potentially nuked them out of the backend if they're not being used at all.  LIke we're querying 'status' on almost every query but I'm not sure they're all used.  Doing an audit pass on that would be nice.

Reply:
That sounds like a very therepautic maintenance issue!

Reply:
I *think* we might even be able to use the type system for it.  If we take the field out of the query, it'll be removed from the generated typestubs, which if they're being access will be cause by tsc:lint, I think?

Reply:
Oh yeah it totally works.

Reply:
just wanted to say i love the gif